Tinfoil-for-Twitter/src/main/AndroidManifest.xml

```diff
@@ -90,6 +90,12 @@
                 <data
                     android:host="apps.twitter.com"
                     android:scheme="https"/>
+                <data
```

Comment:
I see below 2 fb.com addresses snuck in, I'll repush and remove these.

The permissions did not conflict, so I didn't see FINE_LOCATION added. That is suspicious and should not be in there. It is part of the merge, so I think removing it as a new commit would be the sanest approach. The WRITE_EXTERNAL is in there because Tinfoil-Facebook added a way to save photos locally by long pressing them. However it doesn't work on Tinfoil-Twitter (this is also why picasso is added as a gradle dependency)